app/config/services/services.xml

```diff
@@ -171,5 +171,32 @@
 
         <!-- History -->
         <service id="AppBundle\History\EmailSubscriptionHistoryHandler" />
+
+        <!-- Sync with API -->
+
+        <!-- User subscriber -->
+        <service id="AppBundle\Membership\EventListener\UserSubscriber">
+            <argument key="$producer" type="service" id="old_sound_rabbit_mq.sync_api_producer"/>
```

Comment:
It's a question, why not using bind to bind service old_sound_rabbit_mq.sync_api_producer to param name $producer ?

Reply:
This is very fragile/error prone for such a service to use bindings. We could quickly make some mistakes without noticing anything.
Without bind It's explicit, easy to read, easy to understand.

Reply:
Last thing, we are using `$producer` variable in lots of places but they point to different classes most of the time.

Reply:
I'm agree with @B-Galati, it's too overkill for this case.